packages/wdio-utils/src/utils.js

```diff
@@ -48,7 +48,9 @@ export function overwriteElementCommands(propertiesObject) {
  */
 export function commandCallStructure (commandName, args) {
```

Comment:
`commandCallStructure` only changes how args are printed in logs?

Reply:
yes